frontend/src/app/components/ModalExtraConfirm.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
```

Comment:
Would you be able to update frontend/src/app/components/Modal.js to accommodate the extra confirm logic?
The two components seem to share a lot of functionality

Reply:
Sure